packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -870,38 +877,42 @@ async function exportRoleMappings(managementAPIClient, allTeamsData, teamUid, de
       for (const stack of team?.stackRoleMapping ?? []) {
         const roleData = await mapRoleWithTeams(managementAPIClient, stack, team?.name, team?.uid);
         stackRoleWithTeamData.push(...roleData);
-        if(roleData[0]['Stack Name']==='') {
+        if (roleData[0]['Stack Name'] === '') {
           flag = true;
           stackNotAdmin.push(stack.stackApiKey);
         }
       }
     }
   }
-  if(stackNotAdmin?.length) {
-    cliux.print(`warning: Admin access denied to the following stacks using the provided API keys. Please get in touch with the stack owner to request access.`,{color:"yellow"});
-    cliux.print(`${stackNotAdmin.join(' , ')}`,{color:"yellow"});
+  if (stackNotAdmin?.length) {
+    cliux.print(
+      `warning: Admin access denied to the following stacks using the provided API keys. Please get in touch with the stack owner to request access.`,
+      { color: 'yellow' },
+    );
+    cliux.print(`${stackNotAdmin.join(' , ')}`, { color: 'yellow' });
   }
-  if(flag) {
+  if (flag) {
     let export_stack_role = [
       {
         type: 'list',
         name: 'chooseExport',
         message: `Access denied: Please confirm if you still want to continue exporting the data without the { Stack Name, Stack Uid, Role Name } fields.`,
         choices: ['yes', 'no'],
         loop: false,
-      }]
-      const exportStackRole = await inquirer
+      },
+    ];
+    const exportStackRole = await inquirer
       .prompt(export_stack_role)
-      .then(( chosenOrg ) => {
-        return chosenOrg
+      .then((chosenOrg) => {
+        return chosenOrg;
```

Comment:
[MINOR] This .then method is doing nothing at this point of time as I do not see any logic init. We can remove it